python-312/README.md

```diff
@@ -136,6 +138,37 @@ Pythonista!
 
 In this example, you can see how the new implementation of f-strings allows you to include backslashes in embedded expressions. This wasn't possible with f-strings in earlier versions of Python. 
 
+### Static Typing Improvements
+
+You'll find all static typing examples inside the [`typing/`](typing/) directory. You should install the Pyright type checker for PyPI:
+
+```console
+$ python -m pip install pyright
+```
+
+You can then run type checks by running `pyright`. For some features you need to specify `--pythonversion 3.12`.
+
+#### Type Variables and Generic Classes, Functions, and Type Aliases
+
+You can find comparisons between the old and the new syntax for type variables in the following files:
+
+- [`generic_queue.py`](typing/generic_queue.py) and [`generic_queue_311.py`](typing/generic_queue_311.py)
+- [`list_helpers.py`](typing/list_helpers.py) and [`list_helpers_311.py`](typing/list_helpers_311.py)
+- [`concatenation.py`](typing/concatenation.py) and [`concatenation_311.py`](typing/concatenation_311.py)
+- [`inspect_string.py`](typing/inspect_string.py) and [`inspect_string_311.py`](typing/inspect_string_311.py)
+- [`deck.py`](typing/deck.py) and [`deck_311.py`](typing/deck_311.py)
+- [`alias.py`](typing/alias.py) and [`alias_311.py`](typing/alias_311.py)
```

Comment:
:warning: **Fact Check**

Missing files? I'm not seeing the `*_311.py` files in this pull request.

Reply:
Ah, sorry. I had updated the README to remove these links after I combined the 3.11 and 3.12, but the README wasn't committed. Should be fixed now.